vendor/k8s.io/klog/v2/README.md

```diff
@@ -23,6 +23,18 @@ Historical context is available here:
  * https://groups.google.com/forum/#!msg/kubernetes-sig-architecture/wCWiWf3Juzs/hXRVBH90CgAJ
  * https://groups.google.com/forum/#!msg/kubernetes-dev/7vnijOMhLS0/1oRiNtigBgAJ
 
+## Release versioning
+
+Semantic versioning is used in this repository. It contains several Go modules
+with different levels of stability:
+- `k8s.io/klog/v2` - stable API, `vX.Y.Z` tags
+- `k8s.io/tools` - no stable API yet (may change eventually), `tools/v0.Y.Z` tags
```

Comment:
what is k8s.io/tools?

Reply:
It should read "k8s.io/hacks/tools". I have a fix pending in another klog PR for the logcheck tool in that directory.

It's a separate Go module in the same repo.

Reply:
a `tools/v0.Y.Z` tag in k8s.io/klog would match a `k8s.io/klog/tools` module... are we really intending to turn k8s.io/klog into a multi-module repo?

Reply:
It has already been a multi-module repo from the beginning because of "examples". I understand why that was necessary (avoid extra dependencies in the main module).

I wasn't involved in the decision to add "hack/tools", but I guess it was seen as an easier solution than maintaining another repo. So far it hasn't been tagged separately, which is why it is listed without a version in Kubernetes' [hack/tools/go.mod](https://github.com/kubernetes/kubernetes/blob/0ade4678a7ba527d22f6baa81034dea423267608/hack/tools/go.mod#L14). I've already pinged @dims that we should release it separately once we are ready (some changes are pending).

Reply:
> drop k8s.io/kubernetes use of klog's hack/tools/go.mod

For that we first have to move `hack/tools/logcheck` elsewhere. Should that be in a separate repo alongside `k8s.io/klog`, perhaps `k8s.io/logtools`?

/cc @serathius @dims: ^^^

Reply:
if it's a think we really want to release separately, that's much easier to do in a second repo

Reply:
I've opened an issue for further discussion and to track this: https://github.com/kubernetes/klog/issues/312